AraCorrelator/RayTraceCorrelator.h

```diff
@@ -117,6 +119,22 @@ class RayTraceCorrelator : public TObject
             double &arrivalTheta, double &arrivalPhi
         );
         
+        //! function to get lookup the launch angle information
+        /*!
+            \param ant antenna index
+            \param solNum which solution number (0 = direct, 1 = reflected/refracted)
+            \param thetaBin the theta bin desired (bin space, not angle space!!)
+            \param phiBin the phi bin desired (bin space, not angle space!!)
+            \param launchTheta passed by reference, content is replaced with the launch theta angle (in a coordinate where z-axis is along the Earth's radius)
```

Comment:
Advise adding clarifying doc that launch angle is wrt to target, not emitter. Otherwise, looks good to me.

Reply:
The angles in this function are in the station-centric coordinates, not with respect to the target vector.  The launch angle is initially calculated w.r.t. the target position vector in the [RaySolver class in AraSim](https://github.com/ara-software/AraSim/blob/2d7de9db0fa2231a23e487754d8e9b72f9b2f606/RaySolver.cc#L870).  This function uses the same logic as Report::GetAngleAnt, which gets the arrival angles in station-centric coordinates, to get the launch angles in station-centric coordinates.

Reply:
Okay, I see. Maybe make a comment to that effect in the code? But otherwise looks ok.